bundles/org.openhab.core.compat1x/src/main/java/org/openhab/model/sitemap/Chart.java

```diff
@@ -27,6 +27,7 @@
  * @see org.openhab.model.sitemap.SitemapPackage#getChart()
  * @model
  * @generated
+ * @author Kai Kreuzer - Initial contribution
```

Comment:
OK, I was not sure, so they should be added to the list of the excluded bundles.